### DISCUSSION TITLE
Problem with dataset metadata validation - Conformance Class 8

### DISCUSSION THREAD

**Discussion by peterkysel27** [INSPIRE-MIF/helpdesk-validator#1074] [Q&A] [answered]
Hello,

recently we noticed, that in the dataset metadata validation, there is a new Conformance Class 8, which is checking if there is a link for view and download service in the metadata. However, we have a little problem with validation of our metadata.

On this link you can find an example our metadata: https://zbgis.skgeodesy.sk/geoportal/rest/document?id=%7B01DF7E45-7F3E-4541-AC3D-1A0C2AA69E24%7D
We have links for both view service (WMS) and download service (ATOM) + links to directly download the data (see the picture below) We are supplying only the "linkage", "name" and "function" elements.
![image](https://github.com/INSPIRE-MIF/helpdesk-validator/assets/153062514/352cda6f-d99a-4dac-a683-0d10d80b03ef)

However, when we tried to validate the metadata, the result for the Conformance Class 8 was failed for both view and download service.
Here is the validation report: https://inspire.ec.europa.eu/validator/test-run/details.html?id=EID26a33375-89e5-4752-ade5-958eaa1b1298

The only case, when the validation ended with the "passed" result was when we supplied the "protocol" and "applicationProfile" elements. However, according to the technical regulations, these two elements are not mandatory, but without them the validation always ends up with the "failed" result. So should we add these elements into our metadata or it is a bug of the validation?

Thank you for the explanation.
Peter Kysel

**Comment by fabiovinci** [comment, marked as answer]
Dear @peterkysel27,

The CC8 is mandatory if you are implementing the Good Practice "Data and Service Linking Simplification", if this is not the case you can disable it (See the related [discussion ](https://github.com/INSPIRE-MIF/helpdesk-validator/discussions/1009)for more info). 

In any case, fulfilling the additional requirements is recommended to facilitate the linking between the dataset and services.

Hope this helps clarify.

**Comment by peterkysel27** [comment]
Thank you for claryfing. However, I agree with one of the posts in the discussion, that this CC should not be checked defaultly, or at least it should be clearly stated that this CC is not mandatory and it is not part of the monitoring. But we are already implementing changes to apply the Good Practice in our organization, so in the future we should be fine...
